vscode/src/chat/chat-view/ChatsController.ts

```diff
@@ -153,8 +169,8 @@ export class ChatsController implements vscode.Disposable {
                 vscode.commands.executeCommand('cody.chat.focus')
             ),
             vscode.commands.registerCommand('cody.chat.newPanel', async () => {
-                await this.panel.clearAndRestartSession()
-                await vscode.commands.executeCommand('cody.chat.focus')
+                await this.panel?.clearAndRestartSession()
+                this.panel && vscode.commands.executeCommand('cody.chat.focus')
```

Comment:
Is there a reason to not `await` here?
```suggestion
                this.panel && await vscode.commands.executeCommand('cody.chat.focus')
```

Reply:
i've removed this logic because turns out this.panel needs to be initialized for the Login page to work, so i change back to using "when" clause for controlling when to show the sidebars or not